pkg/networkservice/common/monitor/server.go

```diff
@@ -23,6 +23,7 @@ package monitor
 import (
```

Comment:
In current solution we actually dont need changes in network service/monitor pkg. Please revert pkg/networkservice/monitor changes.